src/npc_enchanter.cpp

```diff
@@ -267,24 +267,24 @@ class npc_enchantment : public CreatureScript
         switch (action)
         {
         case 0: //Welcome message on click
-            player->GetSession()->SendAreaTriggerMessage("|cffFF0000Hello there, I will be enchanting your gear!");
+            player->GetSession()->SendAreaTriggerMessage("|cffFF0000Hello there, I will enchant your gear!");
 
             {
-                player->ADD_GOSSIP_ITEM(1, "[Welcome to the enchanting NPC!]", GOSSIP_SENDER_MAIN, 0);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Weapon]", GOSSIP_SENDER_MAIN, 1);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant 2H Weapon]", GOSSIP_SENDER_MAIN, 2);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Shield]", GOSSIP_SENDER_MAIN, 3);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Head]", GOSSIP_SENDER_MAIN, 4);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Shoulders]", GOSSIP_SENDER_MAIN, 5);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Cloak-]", GOSSIP_SENDER_MAIN, 6);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Chest]", GOSSIP_SENDER_MAIN, 7);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Bracers]", GOSSIP_SENDER_MAIN, 8);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Gloves]", GOSSIP_SENDER_MAIN, 9);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Legs]", GOSSIP_SENDER_MAIN, 10);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Feet]", GOSSIP_SENDER_MAIN, 11);
+                player->ADD_GOSSIP_ITEM(1, "|TInterface/ICONS/trade_engraving:24:24:-18|t[Welcome to the enchanting NPC!]", GOSSIP_SENDER_MAIN, 0);
```

Comment:
Why [ ]?

Reply:
It was ugly to leave it without icon on top, so i decide to put Enchanting Icon on it.
Also for that ''Succesfully Enchant text color" replaced red color with modern one, bcoz with red it  looks llike Warning :rofl: 
![Preview](https://user-images.githubusercontent.com/41213210/48573566-e2dfd780-e90c-11e8-8be3-4bf620f5ac58.png)

Reply:
I mean, why the "Welcome" is in [ ]

Reply:
Oh, i'll edit it :+1:

Reply:
Updated [3c4755f](https://github.com/azerothcore/mod-npc-enchanter/pull/1/commits/3c4755f273e0c79631e86699b6c3d7f710dd5c1d)